runtime/compiler/build/toolcfg/aix-xlc/common.mk

```diff
@@ -213,6 +213,10 @@ endif
 
 SOLINK_SLINK+=$(PRODUCT_SLINK) m j9thr$(J9_VERSION) j9hookable$(J9_VERSION)
 
+ifneq (,$(findstring xlclang,$(notdir $(CC))))
```

Comment:
Technically this should be `ifneq (,$(findstring xlclang++,$(notdir $(CXX))))`

Reply:
@pshipton Before implementing this change, the personal build succeeded with no problems. It seems unlikely, but is it possible that this has something to do with it?

Reply:
The PR testing worked fine with this change https://ci.eclipse.org/openj9/job/Pipeline_Build_Test_JDK13_ppc64_aix/43/, it produced 
https://140-211-168-230-openstack.osuosl.org/artifactory/ci-eclipse-openj9/Build_JDK13_ppc64_aix_Personal/36/OpenJ9-JDK13-ppc64_aix-20191018-143648.tar.gz

The subsequent build failure was my fault, I ran more PR testing but incorrectly. The currently running PR test should build. I'm running tests this time, although not sure they are of any value since it seems the static linking isn't working properly yet.